packages/docs/docs/features/previews.mdx

```diff
@@ -0,0 +1,55 @@
+---
+id: previews
+title: Previews
+sidebar_position: 5
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Previews allow you to develop components in complete isolation from the rest of your application.
+
+To use previews first, add `radon-ide` package to your project.
+
+<Tabs groupId="package-managers">
+  <TabItem value="npm" label="NPM">
+
+    npm i radon-ide
+
+  </TabItem>
+  <TabItem value="yarn" label="YARN">
+
+    yarn add radon-ide
```

Comment:
Shouldn't it be a dev dependency?

Reply:
I'm pretty sure it needs to be a dependency 🤔